src/en/clean-copy/03-Part II. The Globe of Belgium/15.md

```diff
@@ -1,65 +1,65 @@
 ### Orange Zest, Coriander, and the Hoegaarden Milkman
 
-Belgium is a small country with a huge beer tradition that spans many centuries in the past. Let's take, for example, the Belgian white beer — *witbier* or *bière blanche*. The history of this wonderful beer style begins in the XV century! Belgium was a part of the Netherlands then, which was a colonial empire that imported lots of exotic goods from overseas territories, including spices.
+Belgium is a small country with a profound beer tradition which spans many centuries. Let's take, for example, Belgian white beer — *witbier* or *bière blanche*. The history of this unique beer style begins in the XV century. Modern-day Belgium was a part of the Netherlands then, which was a mercantile empire that imported lots of exotic goods from overseas territories, including spices.
 
-All beer was sour those days, and in order to get rid of this sourness, Belgian monastic brewers from the village of Hoegaarden started experimenting with these spices, particularly, coriander and famous oranges from the island of Curaçao. That's how the famous witbier recipe of 1445 was born, and closer to the 18th century Hoegaarden became one vast brewery.
+Most beer was sour in those days, and in order to get rid of this sourness Belgian monastic brewers from the village of Hoegaarden started experimenting with these spices. In particular, they utilized coriander and oranges from the island of Curaçao. This explains in part the origin of how the famous witbier recipe of 1445 was born. As time passed and tradition grew, by the 18th century Hoegaarden became one vast brewery.
 
-Despite *witbier* production stopped in the middle-20th century (in 1957, the last remaining manufacturer — the Tomsin brewery — was closed), the villagers did not allow the recipe to vanish: a local milkman, Pierre Celis, restored the tradition in 1965[ref](http://web.archive.org/web/20211110232136/https://hoegaarden.com/the-history/).
+Despite *witbier* production having all but stopped in the mid-20th century (in 1957, the last remaining manufacturer — the Tomsin brewery — was closed), the villagers did not allow the recipe to vanish. A local milkman, Pierre Celis, restored the tradition in 1965[ref](http://web.archive.org/web/20211110232136/https://hoegaarden.com/the-history/).
 
-#### How to try
+#### Which to Try?
```

Comment:
Are you sure about that?
The intent is rather "How to try [those historical beers today]" than "Which [beers] to try".

Reply:
We we say something like "How and where to find these beers today."

To try would sound a bit odd.

Reply:
That's a repeated section in every chapter. I would prefer something laconic